src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+@use "../utils/variables" as *;
+@use "../utils/extends";
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid $border-color-light;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+
+    margin: 0 auto 40px;
+    height: 134px;
+    width: 160px;
+
+    // transition: transform 0.4s;
+
+    // &:hover {
+    //   box-sizing: border-box;
+    //   border: 3px solid $border-color-light;
+    //   border-radius: 5px;
+    //   transform: scale(1.2, 1.2);
+    // }
```

Comment:
better cleanup the commented out code unless there's a solid reason to keep it

Reply:
@meforoma  This was for the additional task. I just commented it out because it wasn't part of the original task and used it afterwards